grpc/src/client/load_balancing/test_utils.rs

```diff
@@ -234,7 +244,7 @@ impl LbPolicyBuilder for StubPolicyBuilder {
         &self,
         _config: &ParsedJsonLbConfig,
     ) -> Result<Option<LbConfig>, Box<dyn Error + Send + Sync>> {
-        todo!("Implement parse_config in StubPolicyBuilder")
+        todo!("Implement parse_config in StubPolicy");
```

Comment:
Hmm ... do we need this change, given that `parse_config` is a method on the builder?